Palaso/PlatformUtilities/Platform.cs

```diff
@@ -76,7 +77,57 @@ private static string UnixName
 			}
 		}
 
+
+		public static bool IsCinnamon
+		{
+			get
+			{
+				if (!IsUnix)
+					return false;
+				return SessionManager.StartsWith("/usr/bin/cinnamon-session");
```

Comment:
`IsUnix && SessionManager.Starts...`

`IsWasta` could be moved up next to this property.